src/styles/constants/shared.scss

```diff
@@ -0,0 +1,14 @@
+// Theme colors
```

Comment:
Can we rename these to include the general color? For example , $primary1-dark-red-color.

Reply:
Can we forego this change for this branch until we have a clearer idea of how we want to structure colors, variable names, and whether we will be using `CSS IN JS` ?

Reply:
Yup, I think it's fine to defer until we have a better idea of how we want to do things.